packages/common/README.md

```diff
@@ -0,0 +1,17 @@
+# @moj-bichard7/common
+
+- [@moj-bichard7/common](#moj-bichard7common)
+  - [What is the package?](#what-is-the-package)
+  - [Development](#development)
+    - [When do I need to build this?](#when-do-i-need-to-build-this)
+
```

Comment:
```suggestion

## Contents

- [What is the package?](#what-is-the-package)
- [Development](#development)
  - [When do I need to build this?](#when-do-i-need-to-build-this)

```

Reply:
I have an auto-generate table of contents feature in my editor. I can remove that top level if you like.

Reply:
Yes please, I think the top level one is just noise and adds unnecessary extra indentation.